We.Sell.Bread.API.Unit.Tests/BaseFixture.cs

```diff
@@ -6,12 +6,15 @@ namespace We.Sell.Bread.API.Unit.Tests.Tests
     public class BaseFixture
     {
         public readonly ILogger<CustomerController> Logger;
+        public readonly ILogger<ProductController> iLogger;
 
         public BaseFixture() 
         {
             var mock = new Mock<ILogger<CustomerController>>();
+            var imock = new Mock<ILogger<ProductController>>();
 
             Logger = mock.Object;
+            iLogger = imock.Object;
```

Comment:
we don't need another instance of the logger, we can re-use the one that's there in all our tests